### ISSUE TITLE
missing generated.__Directive method

### ISSUE THREAD

**Issue by Fruchtgummi** [99designs/gqlgen#1543]
gqlen version 0.13.0

I have a error since the last generate and i can't fix it!:

```
.\services.go:46:3: cannot use s (type *Server) as type generated.ResolverRoot in field value:
	*Server does not implement generated.ResolverRoot (missing generated.__Directive method)
```

I understand that...

**Comment by hiepvv** [comment]
Try to remove your gqlen from your go pkg, go bin and run go get gql again. Hope this help

**Comment by macnibblet** [comment]
I'm also having this problem, tried the solution above, still no success.

**Comment by macnibblet** [comment]
Looks like some issues with gqlparser v2.2.0 so downgrading to 2.1.0 works

```
$ go get github.com/vektah/gqlparser/v2@v2.1.0
go get: downgraded github.com/vektah/gqlparser/v2 v2.2.0 => v2.1.0
```

**Comment by turkenh** [comment]
@macnibblet your suggestion worked for me too, thank you!

**Comment by lwc** [comment]
Fixed in v0.14.0
